packages/clerk-js/src/ui/elements/LineItems.tsx

```diff
@@ -80,7 +81,7 @@ function Group({ children, borderTop = false, variant = 'primary' }: GroupProps)
  * -----------------------------------------------------------------------------------------------*/
 
 interface TitleProps {
-  title: string | LocalizationKey;
+  title?: string | LocalizationKey;
```

Comment:
title on the title component should not be optional. This is similar to the not about this update not being a line item, and more of a notice that gets displayed above the line items.